src/plugins/intel_gpu/src/graph/data.cpp

```diff
@@ -103,7 +103,7 @@ void data_inst::load(BinaryInputBuffer& ib) {
             _outputs[0]->copy_from(get_network().get_stream(), _buf.data());
         }
 
-        ib.addConstData(id(), _outputs[0]);
+        ib.addConstData(get_network_id(), id(), _outputs[0]);
```

Comment:
Does it always copy constant for all networks not created for dynamic batch? i.e., networks created for multiple streams, not for the dynamic batches..

Reply:
For the case of multiple streams, const data is shared among them by L90.